### PR TITLE
Add Team Carousel to the Team Page

### PR DESCRIPTION
**Pull request by IcedTea2K**
- Revamp the ProfileCard
- Add a carousel which uses fetched data
image

### CODE REVIEW

**Comment by Kevin-Tan25** [comment]
Some of the pictures look squished is there any way to set 10 x 10 rem for img width and height

File: src/components/shared/team/ProfileCard.jsx [hunk 1]
```diff
@@ -4,13 +4,14 @@ import './/profileCard.css';
...
     <>
-      <div className='card_container'>
-        <div className='card_information_container'>
-          <div>
-            <img className='profile_img' src={img} alt='name' />
-          </div>
-          <div className='name_title'>{name}</div>
-          <div className='exec_title'>{title}</div>
+      <div className= 'flex w-[23.8rem] h-[13.0rem] bg-[rgb(43,57,80)] rounded-[1.25rem] px-[0.8rem] py-[1.5rem] gap-[1.25rem]'>
+        <div className='w-[9.8rem] h-[10.1rem] rounded-[5.7rem] overflow-hidden'>
```
Comment (Kevin-Tan25) [inline src/components/shared/team/ProfileCard.jsx]: hmm is there anything else we could try to make the image fit?
I just generated this off GPT
```
<div class="aspect-w-1 aspect-h-1 overflow-hidden">
  <img src="path/to/your/image.jpg" alt="Face" class="object-cover object-center w-full h-full" />
</div>
```
if not then we can make sure that the images going in have an aspect ratio of 1:1

File: src/components/shared/team/ProfileCard.jsx [hunk 2]
```diff
@@ -4,13 +4,14 @@ import './/profileCard.css';
...
+          <img className='object-fill w-full h-full' src={img}></img>
+        </div>
+        <div className='h-[10.1rem] bg-transparent flex flex-col gap-[2.3rem]'>
+          <h3 className='text-h3 font-header bg-transparent'>{name}</h3>
+          <p className='font-content bg-transparent text-[0.81rem]'>Title: {title}</p>
+          <p className='font-content bg-transparent text-[0.81rem]'>Email: </p>
```
Comment (Kevin-Tan25) [inline src/components/shared/team/ProfileCard.jsx:14]: also just double checking these emails would be per department? should we remove the email field and instead have a block at top of each department for this information e.g. tech department at the top we have a block of area where we can display email

File: src/components/shared/team/ProfileCard.jsx (hunk 1 above)
Comment (IcedTea2K) [reply src/components/shared/team/ProfileCard.jsx]: Sorry for the late reply, I'll try that out

File: src/components/shared/team/ProfileCard.jsx (hunk 2 above)
Comment (IcedTea2K) [reply src/components/shared/team/ProfileCard.jsx:14]: Oh, I didn't know that it'd be per department. We could also display common email for each department, but we should probably ask Ayden to design it. 'Cause I don't really know where it should go exactly

**Review by Kevin-Tan25 — APPROVED** [review]

**Pull request merged**